pvlib/irradiance.py

```diff
@@ -1158,24 +1159,24 @@ def perez(surface_tilt, surface_azimuth, dhi, dni, dni_extra,
     F2c = np.vstack((F2c, nans))
 
     F1 = (F1c[ebin, 0] + F1c[ebin, 1] * delta + F1c[ebin, 2] * z)
-    F1 = np.maximum(F1, 0)
+    F1 = nanmaximum(F1, 0)
```

Comment:
I think this function would need a different name, since it is not ignoring nans but propagating them.
However you could also try this instead:
`np.clip(F1, 0, None, out=F1)`

Reply:
Shouldn't be here - git mistake on my end.